lib/inferno/apps/cli/execute/plain_outputter.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'console_outputter'
+
+module Inferno
+  module CLI
+    class Execute
+      # @private
+      class PlainOutputter < ConsoleOutputter
+        # override to disable spinner
+        def print_around_run(_options)
+          puts 'Running tests. This may take a while...'
+          yield
+        end
+
+        def print_error(_options, exception)
+          puts "Error: #{exception.full_message(highlight: false)}"
+          puts exception.backtrace&.join('\n')
```

Comment:
Why is this different from the console outputter version? Also, doesn't `full_message` already include the stack trace?

Reply:
Plain outputter needs `highlight: false` or it outputs ANSI code `\e[1` (to make it bold).

But the backtrace part was redundant so I'm removing that.